### ISSUE TITLE
Trailing double quote removed from cell data

### ISSUE THREAD

**Issue by hugovk** [thombashi/pytablewriter#49]
Python 3.6.12
Python 3.9.7
pytablewriter==0.61.0 (newest)

And reproducible as far back as:
pytablewriter==0.47.0

Modifying the example from https://pytablewriter.readthedocs.io/en/latest/pages/examples/basic.html to add a cell containing `"Debian 11 \"Bullseye\""`:

```python
import pytablewriter as ptw


def main():
    writer = ptw.MarkdownTableWriter()
    writer.table_name = "zone"
    writer.headers = ["zone_id", "country_code", "zone_name"]
    writer.value_matrix = [
        ["1", "AD", "Debian 11 \"Bullseye\""],
        ["2", "AE", "Asia/Dubai"],
        ["3", "AF", "Asia/Kabul"],
        ["4", "AG", "America/Antigua"],
        ["5", "AI", "America/Anguilla"],
    ]

    writer.write_table()


if __name__ == "__main__":
    main()
```

# Actual result

```md
# zone
|zone_id|country_code|     zone_name     |
|------:|------------|-------------------|
|      1|AD          |Debian 11 "Bullseye|
|      2|AE          |Asia/Dubai         |
|      3|AF          |Asia/Kabul         |
|      4|AG          |America/Antigua    |
|      5|AI          |America/Anguilla   |
```

# Expected result

Something like:

```md
# zone
|zone_id|country_code|     zone_name      |
|------:|------------|--------------------|
|      1|AD          |Debian 11 "Bullseye"|
|      2|AE          |Asia/Dubai          |
|      3|AF          |Asia/Kabul          |
|      4|AG          |America/Antigua     |
|      5|AI          |America/Anguilla    |
```

# More info

Same issue with:

```python
        ["1", "AD", 'Debian 11 "Bullseye"'],
```

But it works with single quotes:

```python
        ["1", "AD", "Debian 11 'Bullseye'"],
```
And:
```python
        ["1", "AD", 'Debian 11 \'Bullseye\''],
```

```md
# zone
|zone_id|country_code|     zone_name      |
|------:|------------|--------------------|
|      1|AD          |Debian 11 'Bullseye'|
|      2|AE          |Asia/Dubai          |
|      3|AF          |Asia/Kabul          |
|      4|AG          |America/Antigua     |
|      5|AI          |America/Anguilla    |
```

**Comment by thombashi** [comment]
@hugovk 
Thank you for your report.

I believe the problem had fixed at `pytablewriter 0.63.0`.
Please upgrade and try to re-execute the program.

**Comment by hugovk** [comment]
Confirmed working with `pytablewriter 0.63.0`. Thank you very much!
